src/api/rest/server/server.go

```diff
@@ -134,7 +134,7 @@ func ApiServer() {
 	g.GET("/:nsId/benchmarkall/mcis/:mcisId", rest_mcis.RestGetAllBenchmark)
 
 	g.POST("/:nsId/monitoring/install/mcis/:mcisId", rest_mcis.RestPostInstallMonitorAgentToMcis)
-	
+	g.GET("/:nsId/monitoring/mcis/:mcisId/metric/:metric", rest_mcis.RestGetMonitorData)
```

Comment:
Are these APIs for CB users, or internal APIs only for CB-Dragonfly?
1. The case that these APIs are for CB users
   - Who would be appropriate to provide `InstallMonitorAgentToMcis` and `GetMonitorData` APIs, Tumblebug or Dragonfly?
2. The case that these APIs are internal APIs only for CB-Dragonfly
   - Then it is reasonable that Tumblebug provides these APIs.